### PR TITLE
DAH-2464 display for rank in tiered case

### PR DESCRIPTION
**Pull request by tallulahkay**
DAH-2464

### CODE REVIEW

File: app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js [hunk 1]
```diff
@@ -12,6 +12,22 @@ const getRank = (prefKey, prefLotteryRank) => {
...
+  // Override the key we display for certain cases
+  if (application.preference_name === 'General') {
+    return 'General'
+  } else if (application.preference_name?.includes('Right to Return')) {
+    return 'RtR'
+  } else if (application.preference_name?.includes('Tier')) {
+    return application.custom_preference_type.replace('G', 'General')
+  }
+
+  // TODO: DAH-1904 - clean up references to unused prefrenece type field
```
Comment (cliu02) [inline app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js]: @tallulahkay [DAH-1904](https://sfgovdt.jira.com/browse/DAH-1904) is marked as Closed, does it have a follow up ticket?

Comment (tallulahkay) [reply app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js]: idk... I tagged alula @alulabeshue-sfgov and @fwextensions on the ticket to ask them

**Review by cliu02 — APPROVED** [review]
Looks good, strings in lease up are updated! Non-blocking note: especially with the longer rank name, the checkmark display could use adjustment in another ticket. 
Screenshot 2024-08-05 at 3 55 46 PM

File: app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js (hunk 1 above)
Comment (tallulahkay) [reply app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js]: I'll update and remove the comment referencing that ticket

**Pull request merged**